tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -106,128 +118,87 @@ function EditProfile() {
     )
     setErrors(formValidation.errors)
     setTouched(formValidation.touched)
-    setTimeout(() => errorRef.current.focus(), 0)
+
+    if (!Object.values(formValidation.errors).length) {
+      return dispatch(requestAccess(profileInfo))
+    }
+    return setTimeout(() => errorRef.current.focus(), 10)
+  }
+
+  if (requestedAccess) {
```

Comment:
There was a line here causing a test to fail `setTimeout(() => errorRef.current.focus(), 10)`. Removing it caused the tests to pass, I want to be sure this is correct.

Reply:
Sorry, I'm a little confused. I just pulled it down and that line `setTimeout(() => errorRef.current.focus(), 10)` is present and the tests are passing. That focus is necessary for an accessibility concern that Miles found.

Reply:
On the 231 branch, this was changed to a timeout of 0. I think just making sure that this branch is current with 231 should have everything working correctly.